### PR TITLE
Merging the documents

### LINKED ISSUE #114: Write Requirements for the software

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Fix #114, redo of #123 due to conflict.

### CODE REVIEW

**Comment by Yus147** [comment]
@Huy-Ngo  Is there anything reusable from the Overall Requirement in my branch in #123

File: docs/source/req/use-case/index.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Use-case model
+==============
+
+This section includes the list of use cases we expected to have in the system.
+Since the number of use cases is large, we divide the use cases to smaller groups
+for clarity.
+
+.. toctree::
+   :maxdepth: 2
+
```
Comment (Yus147) [inline docs/source/req/use-case/index.rst:10]: The headings for the 3 use-case model file should be renamed accordingly, right now they look kinda confusing when we take a look at System Requirement
Screenshot from 2021-01-15 18-26-07

Comment (Huy-Ngo) [reply docs/source/req/use-case/index.rst:10]: Good catch. Thanks, I didn't notice that.

**Review by Yus147 — APPROVED** [review]

**Comment by Huy-Ngo** [comment]
Thanks.

**Pull request merged**